openshift/sso74-x509-postgresql-secrets.yaml

```diff
@@ -46,7 +46,7 @@ parameters:
 #  generate: expression
 #  from: super-[a-zA-Z0-9]{6}
 - description: Password of the superuser account for initialization.
-  displayName: Superuser Passsword
```

Comment:
![sss](https://media.giphy.com/media/QtZKO7mb7ebpC/source.gif)

whats wrong with thissss spppellingggss

Reply:
lol still missed one here. For next PR let's refer back to this one and fix anything that's not done yet.

Reply:
just to be clear, it's a bit too late to change it now as this will trigger a recreate deployment. So we'll include those remaining issue next time.